services/web/server/src/simcore_service_webserver/login/_registration_handlers.py

```diff
@@ -45,10 +45,16 @@ async def request_product_account(request: web.Request):
     body = await parse_request_body_as(AccountRequestInfo, request)
     assert body.form  # nosec
 
+    # gets ipinfo
+    peername = (
+        request.transport.get_extra_info("peername") if request.transport else None
+    )
+    ipinfo = f"https://ipinfo.io/{peername}/json" if peername else "Unknown IP address"
```

Comment:
peername?? what is this?
just that you know, Traefik is also configured to transmit the client original IP via ```X-Forwarded-For``` header... for example: https://monitoring.osparc-master.speag.com/whoami or https://osparc-master.speag.com/whoami

Reply:
<img width="659" alt="image" src="https://github.com/ITISFoundation/osparc-simcore/assets/32402063/85a31ba2-c886-4885-bc64-a67812cee54e">

Good point about the header. I will use that as well